setup.py

```diff
@@ -112,7 +150,39 @@ def runtest(self):
         cellprofiler.__main__.stop_cellprofiler()
 
         sys.exit(errno)
-
+        
+class CPPy2Exe(py2exe.build_exe.py2exe):
```

Comment:
Unfortunately, I think it can't be done with data_files in the setup dictionary. The `install` command would see the matplotlib data files, for instance, and install them (and I think it would install them in the root directory of the python installation rather than in site-packages. The install command is rerun as part of py2exe.run and it puts the data files in the dist directory. If the install command is run alone, the data files are not installed. If you run `python setup.py py2exe install`, it will probably do the wrong thing and process data files intended only for py2exe - technically, I should probably remove the data files I added to the distribution.

It might be worth trying package_data (and that would unify py2app and py2exe) - you may be able to specify package data for packages you aren't installing?